src/app/kbv/steps.js

```diff
@@ -29,142 +29,150 @@ module.exports = {
     controller: selfAssessmentRouterController,
     next: [
       {
-        field: "selfAssessmentRouter",
+        field: APP.FIELDS.SELF_ASSESSMENT_ROUTER,
         value: "sa100",
-        next: "/kbv/question/enter-pensions-benefits-self-assessment",
+        next: "question/enter-pensions-benefits-self-assessment",
       },
       {
-        field: "selfAssessmentRouter",
+        field: APP.FIELDS.SELF_ASSESSMENT_ROUTER,
         value: "sa200",
-        next: "/kbv/question/enter-pensions-benefits-short-tax-return",
+        next: "question/enter-pensions-benefits-short-tax-return",
       },
     ],
   },
-  "/question/enter-pensions-benefits-self-assessment": {
-    backLink: null,
-    template: "pensions-benefits-self-assessment",
-    fields: [
-      "statePension",
-      "otherPension",
-      "employmentAndSupportAllowance",
-      "jobSeekersAllowance",
-      "statePensionAndBenefits",
-    ],
-    controller: SelfAssessmentTaxReturnQuestionController,
-    next: "load-question",
-  },
-  "/question/enter-pensions-benefits-short-tax-return": {
+  [`${APP.PATHS.QUESTION}${APP.PATHS.ENTER_PENSION_BENEFITS_SELF_ASSESSMENT}`]:
+    {
+      backLink: null,
+      template: "pensions-benefits-self-assessment",
+      fields: [
+        APP.FIELDS.STATE_PENSION,
+        APP.FIELDS.OTHER_PENSION,
+        APP.FIELDS.EMPLOYMENT_AND_SUPPORT_ALLOWANCE,
+        APP.FIELDS.JOB_SEEKERS_ALLOWANCE,
+        APP.FIELDS.STATE_PENSION_AND_BENEFITS,
+      ],
+      controller: SelfAssessmentTaxReturnQuestionController,
+      next: "load-question",
+    },
+  [`${APP.PATHS.QUESTION}${APP.PATHS.ENTER_PENSION_BENEFITS_SHORT_TAX_RETURN}`]:
+    {
+      backLink: null,
+      template: "pensions-benefits-short-tax-return",
+      fields: [
+        APP.FIELDS.STATE_PENSION_SHORT,
+        APP.FIELDS.OTHER_PENSION_SHORT,
+        APP.FIELDS.EMPLOYMENT_AND_SUPPORT_ALLOWANCE_SHORT,
+        APP.FIELDS.JOB_SEEKERS_ALLOWANCE_SHORT,
+        APP.FIELDS.STATE_PENSION_AND_BENEFITS_SHORT,
+      ],
+      controller: SelfAssessmentTaxReturnQuestionController,
+      next: "load-question",
+    },
+  [`${APP.PATHS.QUESTION}${APP.PATHS.ENTER_RECENT_SELF_ASSESSMENT_PAYMENT}`]: {
     backLink: null,
-    template: "pensions-benefits-short-tax-return",
+    template: "self-assessment-payment",
     fields: [
-      "statePensionShort",
-      "otherPensionShort",
-      "employmentAndSupportAllowanceShort",
-      "jobSeekersAllowanceShort",
-      "statePensionAndBenefitsShort",
+      APP.FIELDS.SELF_ASSESSMENT_PAYMENT_DATE,
+      APP.FIELDS.SELF_ASSESSMENT_PAYMENT_AMOUNT,
     ],
-    controller: SelfAssessmentTaxReturnQuestionController,
-    next: "load-question",
-  },
-  "/question/enter-recent-self-assessment-payment": {
-    backLink: null,
-    template: "self-assessment-payment",
-    fields: ["selfAssessmentPaymentDate", "selfAssessmentPaymentAmount"],
     controller: selfAssessmentPaymentQuestionController,
     next: "load-question",
```

Comment:
Can this use the enum?
see also the ones above this and `answer-security-questions` below